v0/destinations/google_ads_enhanced_conversion/util.js

```diff
@@ -0,0 +1,148 @@
+const { get, set } = require("lodash");
+const sha256 = require("sha256");
+const { httpSend } = require("../../../adapters/network");
+const { isHttpStatusSuccess } = require("../../util/index");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+const ErrorBuilder = require("../../util/error");
+const {
+  DISABLE_DEST,
+  REFRESH_TOKEN
+} = require("../../../adapters/networkhandler/authConstants");
+const Cache = require("../../util/cache");
+
+const conversionActionIdCache = new Cache(24 * 60 * 60);
+
+const {
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const { BASE_ENDPOINT } = require("./config");
+
+/**
+ *  This function is used for collecting the conversionActionId using the conversion name
+ * @param {*} method
+ * @param {*} headers
+ * @param {*} params
+ * @returns
+ */
+
+const getConversionActionId = async (method, headers, params) => {
```

Comment:
Did we test out this functionality? 
 - Maybe set a lower cache expiry and tets it out?

Reply:
No